docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
+    </tr>          
+    <tr>
+      <td><b>min_by</b>(<i>expression1, expression2</i>)</td>
+      <td>any numeric, string, date/time or arrays of these types</td>
+      <td>Returns the value of expression1 associated with the minimum value of expression2.</td>
+    </tr>      
+    <tr>
+      <td><b>percentile</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>numeric Type, double, integral type</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the exact percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>percentile</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric type; double; integral type</td>
+      <td>Percentage array is an array of number between 0 and 1; frequency is a positive integer. Returns the exact percentile value array of numeric expression at the given percentage(s).</td>
+    </tr>        
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the approximate percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the approximate percentile value of numeric expression at the given percentage.</td>
+    </tr>             
+    <tr>
+      <td><b>skewness</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the skewness value calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>{stddev_samp | stddev | std}</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the sample standard deviation calculated from values of a group.</td>
+    </tr>  
+    <tr>
+      <td><b>stddev_pop</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the population standard deviation calculated from values of a group.</td>
+    </tr>
+    <tr>
+      <td><b>sum</b>(<i>expression</i>)</td>
+      <td>numeric</td>
+      <td>Returns the sum calculated from values of a group.</td>
+    </tr>       
+    <tr>
+      <td><b>{variance | var_samp}</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the sample variance calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>var_pop</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the population variance calculated from values of a group.</td>
+    </tr>        
+  </tbody>
+</table>
+
+### Examples
+{% highlight sql %}
+--base table 
```

Comment:
nit: need a space after `--`

Reply:
`-- A test table used in the following examples`?

Reply:
sure